src/modules/workflowsettings/Elsa.WorkflowSettings/Services/WorkflowSettings/WorkflowSettingsManager.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Elsa.WorkflowSettings.Abstractions.Providers;
+using Elsa.WorkflowSettings.Abstractions.Services.WorkflowSettings;
+using Elsa.WorkflowSettings.Models;
+
+namespace Elsa.WorkflowSettings.Services.WorkflowSettingsContexts
+{
+    public class WorkflowSettingsManager : IWorkflowSettingsManager
+    {
+        private readonly IEnumerable<IWorkflowSettingsProvider> _workflowSettingsProviders;
+
+        public WorkflowSettingsManager(IEnumerable<IWorkflowSettingsProvider> workflowSettingsProviders)
+        {
+            _workflowSettingsProviders = workflowSettingsProviders;
+        }
+
+        public async ValueTask<WorkflowSetting> LoadSettingAsync(WorkflowSetting workflowSetting, CancellationToken cancellationToken = default)
```

Comment:
When loading a setting, it seems odd to have to provide a setting. Instead, the caller should have to provide only a workflow blueprint ID and key for the setting to load.